lib/cog_api/http/base.ex

```diff
@@ -52,7 +52,8 @@ defmodule CogApi.HTTP.Base do
 
   def find_id_by(%Endpoint{}=endpoint, resource, find_fun)
       when is_function(find_fun) do
-    with {:ok, %{^resource => items}} <- get(endpoint, resource) do
```

Comment:
I realize this it outside the scope of this PR. I just noticed because there was a change here. I still think it would be worth deleting these if we can.